dockstore-webservice/src/main/resources/migrations.1.6.0.xml

```diff
@@ -191,4 +191,10 @@
         <addForeignKeyConstraint baseColumnNames="validationid" baseTableName="version_validation" constraintName="fkvalidationid" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="validation"/>
     </changeSet>
 
+    <changeSet id="AddNotNullAbsolutePathConstraint" author="coverbeck">
+        <sql dbms="postgres">
+            UPDATE sourcefile SET absolutepath=path WHERE absolutepath isnull
```

Comment:
Should 'isnull' have a space? Or are both 'is null' and 'isnull' valid?

Reply:
DataGrip suggested `isnull` and it seems to work, and i picked it without thinking. I have seen `is null` more commonly though, and there are other examples of it with the space already in migrations files, so I have changed it in a new commit.